core/modules/system/css/menu-dropdown.theme.css

```diff
@@ -48,14 +48,14 @@
   content: '-';
 }
 
-@media (min-width: 768px) {
+@media (min-width: 48em) {
```

Comment:
Thank you for changing this, I thought that might bug you. This CSS came from the upstream library.